app/.gitignore

```diff
@@ -44,3 +44,6 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+# keystore related
```

Comment:
Changes for keystore are depended on each developer's machine. Merging these changes will break authentication for other devs.